main.go

```diff
@@ -16,11 +16,17 @@ import (
 )
 
 func connectionChecker(peer smtpd.Peer) error {
+	if *allowedSender == "" {
```

Comment:
Shouldn't this have been `*allowedNets`?

This will be fixed in upstream decke/smtprelay#18.